x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { ResultHeaderItem } from './result_header_item';
+import { ResultMeta } from './types';
+
+import './result_header.scss';
+
+interface Props {
+  showScore: boolean;
+  resultMeta: ResultMeta;
+}
+
+export const ResultHeader: React.FC<Props> = ({ showScore, resultMeta }) => {
+  const showEngineLabel: boolean = resultMeta.id !== resultMeta.scopedId;
+
+  return (
+    <div className="appSearchResultHeader">
```

Comment:
Semantic markup cont'd - I think the `header` tag fits this use case well
```suggestion
    <header className="appSearchResultHeader">
```